src/style.css

```
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300px;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
```

Do not use tags in styles. Add class to tag and use it
```suggestion
.body {
```